src/sentry/static/sentry/less/group-detail.less

```diff
@@ -374,63 +374,6 @@
   }
 }
 
-/**
-* Event Toolbar
-* ============================================================================
-*/
-
-.event-toolbar {
-  position: relative;
-  margin-bottom: -5px;
-  // z-index seems unnecessary, but increasing (instead of removing) just in case(billy)
-  // Fixes tooltips in toolbar having lower z-index than .btn-group .btn.active
-  z-index: 3;
-  .clearfix;
-  padding: 20px 30px 20px 40px;
-
-  h4 {
-    line-height: 1.3;
-    margin: 0;
-    font-size: 16px;
-
-    .event-id {
-      font-weight: normal;
-    }
-  }
-
-  time {
-    color: @60;
-  }
-
-  .icon-alert {
-    margin-left: 5px;
-    color: @yellow-orange;
-    font-size: 14px;
-  }
-
-  .json-link {
-    margin-left: 8px;
-    padding-left: 8px;
-    position: relative;
-
-    &:before {
-      display: block;
-      position: absolute;
-      content: '';
-      left: 0;
-      top: 2px;
-      height: 14px;
-      border-left: 1px solid @trim;
-    }
-  }
-
-  .tooltip {
-    .tooltip-inner {
-      max-width: 250px;
```

Comment:
Is this no longer relevant?

Reply:
I added it as an inline style to the tooltip, since there were other inline styles already. see line 63.

```
style={{textAlign: 'left', margin: 0, minWidth: '200px', maxWidth: '250px'}}
```